lib/osf-components/addon/components/file-browser/add-new/upload/component.ts

```diff
@@ -26,7 +27,8 @@ export default class Upload extends Component<Args> {
         preventMultipleFiles: false,
         acceptDirectories: false,
         autoProcessQueue: true,
-        autoQueue:true,
+        autoQueue: true,
+        parallelUploads: this.args.manager.rootFolder instanceof GithubProviderFile ? 1 : 2,
```

Comment:
You don't just want to put an parallelUpload property in the File and ProviderFile classes and set them differently for Github? That would match more how we do provider-specific functionality.

Reply:
If you go that route, I would make it an integer rather than a boolean, that way if we wanted to increase the limit for providers that can handle more, we have the option to do that in the future.